group/static/group/profile.js

```diff
@@ -1,17 +1,19 @@
 document.addEventListener('DOMContentLoaded', function() {
 
-    join_btn = document.querySelector('.join_button');
-    request_btn = document.querySelector('.request_button');
+    const join_btn = document.querySelector('.join_button');
+    const request_btn = document.querySelector('.request_button');
+    const viewMember_btn = document.querySelector('.view-members-btn');
+    const delete_btn = document.querySelector('#btn-delete');
+
     if (join_btn) {
         join_btn.onclick = joinOrLeaveGroup;
     }
     if (request_btn) {
         request_btn.onclick = createOrDeleteGroupRequest;
     }
+    viewMember_btn.onclick = showGroupMembers;
 
-    document.querySelectorAll('.view-members-btn').forEach(btn => {
-        btn.addEventListener('click', showGroupMembers);
-    });
+    delete_btn.onclick = deleteGroup;
```

Comment:
don't you need to check if the delete_btn exists first?

Reply:
only import javascript on profile, so tested and it was working okay. 
I think we can actually delete others to check as well, if statements.
unless we are using this javascript others it's good.